doc/api/crypto.md

```diff
@@ -2800,6 +2800,46 @@ threadpool request. To minimize threadpool task length variation, partition
 large `randomFill` requests when doing so as part of fulfilling a client
 request.
 
+### `crypto.randomInt([min, ]max[, callback])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `min` {integer} Start of random range (inclusive). **Default**: `0`.
+* `max` {integer} End of random range (inclusive).
+* `callback` {Function} `function(err, n) {}`.
+
+Return a random integer `n` such that `min <= n <= max`.  This
+implementation avoids [modulo
+bias](https://en.wikipedia.org/wiki/Fisher%E2%80%93Yates_shuffle#Modulo_bias).
+
+The cardinality of the range (`max - min + 1`) must be at most `2^48 -
+1`. `min` and `max` must be safe integers.
+
+If the `callback` function is not provided, the random integer is generated
+synchronously.
+
+```js
+// Asynchronous
+crypto.randomInt(3, (err, n) => {
+  if (err) throw err;
+  console.log(`Random number chosen from (0, 1, 2, 3): ${n}`);
```

Comment:
Just a suggestion, why can't we have promisified function also.

Reply:
@yashLadha ... a Promisified version of the crypto APIs is coming. I expect to be able to open the PR in early September (if not sooner)